### PR TITLE
A skeleton is created and added to the Market screen. 

### PR DESCRIPTION
**Pull request by AndMkh7**
Changed Market screen to display Skeleton effect.

### CODE REVIEW

File: screens/Market/MarketScreen.jsx [hunk 1]
```diff
@@ -1,15 +1,24 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect, useState } from "react";
 import { Text, FlatList, TextInput, TouchableOpacity, View, ScrollView, } from 'react-native';
```
Comment (M7Arman) [inline screens/Market/MarketScreen.jsx:2]: double quote in row 1 and a single quote in row 2. This doesn't make sense

File: screens/Market/MarketScreen.jsx [hunk 2]
```diff
@@ -80,9 +95,12 @@ const MarketScreen = ({ navigation }) => {
           key={item => item.id}
           keyExtractor={item => item.id}
           renderItem={({ item }) => {
-            return <Item item={item} navigation={navigation} />;
+            return  isLoading ? <MarketSkeletonItem/>
+              : <Item item={item} navigation={navigation} />
+              ;
```
Comment (M7Arman) [inline screens/Market/MarketScreen.jsx:100]: I saw this callback in 3 places. Create a callback and reuse

File: components/MarketSkeleton.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+const MarketSkeletonItem = () => {
+  
+
+  return (
+    <SkeletonPlaceholder flexDirection="row" speed={1500} backgroundColor={'#b2cdcf'} highlightColor={'#F2F8FC'}>
+      <SkeletonPlaceholder.Item flexDirection="row">
+
+        <SkeletonPlaceholder.Item flexDirection="column" marginRight={8}>
+          <SkeletonPlaceholder.Item width={windowWidth * 0.29} height={windowWidth * 0.29} borderRadius={8} />
```
[the remark this answers is not in the document]
Comment (AndMkh7) [reply components/MarketSkeleton.jsx:15]: it`s about windowWidth* 0.29 ?

File: screens/Content/ContentScreen.jsx [hunk 4]
```diff
@@ -42,7 +42,7 @@ const ContentScreen = ({ navigation, route }) => {
         <View style={styles.swiperItem}>
 
           <SwiperFlatList
-            autoplay={false}
+            autoplay={true}
```
[the remark this answers is not in the document]
Comment (AndMkh7) [reply screens/Content/ContentScreen.jsx:45]: I'm sorry I didn't notice

File: components/MarketSkeleton.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (M7Arman) [reply components/MarketSkeleton.jsx:15]: yes

[the remark this answers is not in the document]
Comment (AndMkh7) [reply components/MarketSkeleton.jsx:15]: Done !

File: screens/Feed/FeedScreen.jsx [hunk 5]
```diff
@@ -9,8 +9,8 @@ const FeedScreen = ({ navigation }) => {
   const { feedData } = useContext(DNAdataContext);
 
   return (
-    <View style={{ flex: 1, backgroundColor: "white" }}>
-      <View style={{ flex: 1, alignItems: "center" }}>
+    <View style={styles.feedScreen}>
+      <View style={styles.feedScreenContainer}>
```
[the remark this answers is not in the document]
Comment (AndMkh7) [reply screens/Feed/FeedScreen.jsx:13]: Done ! And I was catch this mistake in 2 other screens and fix it. Thanks )

File: screens/Market/MarketScreen.jsx (hunk 1 above)
Comment (AndMkh7) [reply screens/Market/MarketScreen.jsx:2]: Fixed

File: screens/Market/MarketScreen.jsx (hunk 2 above)
Comment (AndMkh7) [reply screens/Market/MarketScreen.jsx:100]: I am looking at several solutions for this situation. Tomorrow morning I will test the most compact and optimal way to solve !

**Pull request merged**